styles.css

```diff
@@ -91,6 +95,351 @@ ul li a {
+  font-size: 36px;
+  line-height: 44px;
+}
+
+.card-1 p,
+.card-2 p,
+.card-3 p,
+.card-4 p,
+.card-5 p,
+.card-6 p {
+  color: #fff;
+  font-family: arial, sans-serif, 'Inter';
+  font-weight: 400;
+  font-size: 16px;
+  width: 100%;
+  padding: 0 24px;
+}
+
+.card-1-ul,
+.card-2-ul,
+.card-3-ul,
+.card-4-ul,
+.card-5-ul,
+.card-6-ul {
+  display: flex;
+  gap: 12px;
+  padding: 5px 24px;
+}
+
+.card-1 .html-btn,
+.card-2 .html-btn,
+.card-3 .html-btn,
+.card-4 .html-btn,
+.card-5 .html-btn,
+.card-6 .html-btn {
+  color: #fff;
+  background: rgba(255, 255, 255, 0.24);
+  border: 1px solid #8993a4;
+  width: 56px;
+  height: 40px;
+  font-weight: 500;
+  font-family: arial, sans-serif, 'Inter';
+  font-size: 15px;
+  line-height: 20px;
+  cursor: pointer;
+  margin: 10px 0;
+}
+
+.card-1 .bs-btn,
+.card-2 .bs-btn,
+.card-3 .bs-btn,
+.card-4 .bs-btn,
+.card-5 .bs-btn,
+.card-6 .bs-btn {
+  background: rgba(255, 255, 255, 0.24);
+  border: 1px solid #8993a4;
+  width: 95px;
+  height: 40px;
+  font-weight: 500;
+  font-family: arial, sans-serif, 'Inter';
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  cursor: pointer;
+  margin: 10px 0;
+}
+
+.card-1 .ruby-btn,
+.card-2 .ruby-btn,
+.card-3 .ruby-btn,
+.card-4 .ruby-btn,
+.card-5 .ruby-btn,
+.card-6 .ruby-btn {
+  background: rgba(255, 255, 255, 0.24);
+  border: 1px solid #8993a4;
+  width: 61px;
+  height: 40px;
+  font-weight: 500;
+  font-family: arial, sans-serif, 'Inter';
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  cursor: pointer;
+  margin: 10px 0;
+}
+
+.card-1 .see-pro,
+.card-2 .see-pro,
+.card-3 .see-pro,
+.card-4 .see-pro,
+.card-5 .see-pro,
+.card-6 .see-pro {
+  color: #fff;
+  font-weight: 700;
+  font-size: 17px;
+  width: 100%;
+  height: 48px;
+  background-color: #ff6b00;
+  border: none;
+  cursor: pointer;
+}
+
+.about-me {
+  padding: 0;
+  background-color: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.basic-info {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  padding-top: 100px;
+  background-image: url("./Img/Group.svg");
+  background-repeat: no-repeat;
+  background-position: bottom;
+  background-position-x: 100%;
+  background-position-y: 90%;
+}
+
+.basic-info h2 {
+  font-size: 72px;
+  font-weight: 400;
+  font-family: arial, sans-serif, 'Crete Round';
+  line-height: 88px;
+  color: #ff6b00;
+  padding: 0 24px;
+}
+
+.basic-info p {
+  font-size: 16px;
+  color: #f4f5f7;
+  font-weight: 400;
+  font-family: arial, sans-serif, 'Inter';
+  line-height: 24px;
+  padding: 0 24px;
+}
+
+.basic-info .rus-btn {
+  width: 161px;
+  height: 48px;
+  background-color: #ff6b00;
+  border: none;
+  cursor: pointer;
+}
+
+.basic-info ul {
+  padding: 24px 0 100px 24px;
+}
+
+.btm-bor {
+  padding: 24px;
+}
+
+.lan-tag,
+.fw-tag,
+.skill-tag {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.rect-55 {
+  width: 12px;
+}
+
+.rect-56 {
+  width: 8px;
+}
```

Comment:
As requested by the previous reviewer, kindly adjust the space between the bullets and the titles and at the end of the section in order to follow the design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/219826349-638f1313-61e7-40a5-baa0-943c6bdc4cf1.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/219826377-a4601d96-6a09-4a84-9fc6-8f600b7e3ddf.png)